include/crow/http_response.h

```diff
@@ -218,6 +218,12 @@ namespace crow
         void set_static_file_info(std::string path)
         {
             utility::sanitize_filename(path);
+            set_static_file_info_unsafe(path);
+        }
+        
+        ///Return a static file as the response body without sanitising the path (use set_static_file_info instead)
```

Comment:
I don't know why clang-format doesn't pick up on this, but there should be a space in here:
```suggestion
        /// Return a static file as the response body without sanitizing the path (use set_static_file_info instead)
```